api/user/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from 'express';
+import { getUserSessionTimeout } from './userTimeout';
+import { getConfigValue } from '../configuration'
+import { SESSION_TIMEOUTS } from '../configuration/references'
+
+interface SessionRequest extends express.Request {
+  session: {
+    user: {
+      roles: ['']
+    }
+  },
+}
+
+// TODO: Need to test error condition here.
+export async function getUserDetails(req: SessionRequest, res: express.Response, next: express.NextFunction) {
```

Comment:
could we not do this please? I never liked the enhancedRequest etc before either. Instead try doing a declaration merging, but only if the @types aren't available

Reply:
Fixed after call with Paddy.